wsi_superpixel_guided_labeling/web_client/views/vue/components/AnnotationOpacityControl.vue

```diff
@@ -0,0 +1,72 @@
+<script>
+import _ from 'underscore';
+
+export default {
+    props: ['activeLearningSetup', 'categories', 'update'],
+    data() {
+        return {
+            opacitySlider: 1.0
+        };
+    },
+    watch: {
+        opacitySlider(opacity) {
+            _.map(this.categories, (cat) => {
+                const newColor = this.rgbStringToArray(cat.strokeColor);
+                cat.strokeColor = `rgba(${newColor}, ${opacity})`;
+                return cat;
```

Comment:
Sounds good! I added this change and also added a small change so that the stroke color is synchronized with the fill color if it is changed (and opacity is still `0`).

One potential concern could be feature visibility though... If the users is lowering opacity of the borders to see something clearer it may be a surprise when the border is suddenly (at least partially) opaque again... Although this may be a non-issue if users are always selecting relatively transparent fill colors.

Reply:
The current behavior is fine -- another approach we could have done would be to have the slider go from opaque to the fill color rather than opaque to transparent and the jump to the fill color.